### ISSUE TITLE
Feature request: make number of `events` requested configurable

### ISSUE THREAD

**Issue by aegershman** [cloudfoundry/bosh#1992]
[When performing `bosh events`, there is a max cap of 200 events retrieved.](https://github.com/cloudfoundry/bosh/blob/master/src/bosh-director/lib/bosh/director/api/controllers/events_controller.rb#L7). 

I would like to be able to configure this to retrieve more than 200 events at a time, similar to how you can pass the number of recent tasks to display as a param with `bosh tasks --recent=some_number`

**Comment by aegershman** [comment]
The more I think about it, the more I think using `events` might not be the best approach to what I actually need to get done.

What I need to do is pull the total number of VMs which were created / destroyed. Every single one. I know this sounds a bit silly, as the total number of VMs churned over doesn't really matter that much. But I could really use the data to impress some of the more, ah, "old school" members of the corporation I work for. I fully understand that in BOSH land, "total VMs churned" is not particularly impressive, but considering there are people I have to debate with who are doing manual VM provisioning/upgrading, to them this seems like an impressive metric & it would be nice to be able to shove it in their face.

tl;dr, this is for demonstrating the value

**Comment by cppforlife** [comment]
i would recommend writing a little script that does that (since we dont
have --follow option for now in the cli), or just going into the database
and running that query.
